bitmap-annashivwill/lib/file-reader.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const fs = require('fs');
+
+const fileReader = module.exports = function(file, callback) {
+  fs.readFile(file, function(err, data) {
+    if (err) return callback(err);
+    return callback(null, data);
+  })
+}
```

Comment:
@William-Skelton @shivprogrammer @annaul awesome job creating a `file-reader` module for reusability